### PR TITLE
Grafana ironbank adder

### PR DESCRIPTION
**Pull request by Kirandeep-Singh-Khehra**
### Description of the change

Addition of image directory and workflows for hardening of `grafana/ironbank` image using docker-compose.

### Checklist

- [x] Added to the `README.md` using [readme-generator](https://github.com/rapidfort/community-images/readme-generator)

### CODE REVIEW

File: community_images/grafana/ironbank/certs/server.key
Comment (AnmolVirdi) [inline community_images/grafana/ironbank/certs/server.key:1]: Please remove certs directory.

File: community_images/grafana/ironbank/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+version: '2'
+
+services:
+  prometheus:
+    image: prom/prometheus:v2.17.1
```
Comment (AnmolVirdi) [inline community_images/grafana/ironbank/docker-compose.yml:5]: is it possible to use Rapidfort's prometheus image here?
https://hub.docker.com/r/rapidfort/prometheus

File: community_images/grafana/ironbank/certs/server.key
Comment (Kirandeep-Singh-Khehra) [reply community_images/grafana/ironbank/certs/server.key:1]: It is removed in the recent commit.

File: community_images/grafana/ironbank/docker-compose.yml (hunk 1 above)
Comment (Kirandeep-Singh-Khehra) [reply community_images/grafana/ironbank/docker-compose.yml]: Yes, we can. I have replaced the line and all the hardening steps are working fine.

**Pull request merged**